src/confcom/azext_confcom/security_policy.py

```diff
@@ -545,12 +545,12 @@ def load_policy_from_arm_template_str(
     aci_list = [
         item
         for item in arm_resources
-        if item["type"] == config.ACI_FIELD_TEMPLATE_RESOURCE_LABEL
+        if item["type"] in config.ACI_FIELD_SUPPORTED_RESOURCES
     ]
 
     if not aci_list:
         eprint(
-            f'Field ["type"] must contain value of ["{config.ACI_FIELD_TEMPLATE_RESOURCE_LABEL}"]'
+            f'Field ["type"] must contain one of "{config.ACI_FIELD_SUPPORTED_RESOURCES}"'
```

Comment:
Nit: stylistically, I would remove the quotes around {config.ACI_FIELD_SUPPORTED_RESOURCES}, since it will come out looking like "['item1'. 'item2']"